package.json

```diff
@@ -198,5 +190,26 @@
       "app": "electron_app"
     },
     "afterSign": "scripts/electron_afterSign.js"
+  },
+  "jest": {
+    "testEnvironment": "jest-environment-jsdom-thirteen",
+    "testMatch": [
+      "<rootDir>/test/**/*-test.js"
+    ],
+    "setupFilesAfterEnv": [
+      "<rootDir>/node_modules/matrix-react-sdk/test/setupTests.js"
+    ],
+    "moduleNameMapper": {
+      "\\.(gif|png|svg|ttf|woff2)$": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/imageMock.js",
+      "\\$webapp/i18n/languages.json": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/languages.json",
+      "^browser-request$": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/browser-request.js",
+      "^react$": "<rootDir>/node_modules/react",
+      "^react-dom$": "<rootDir>/node_modules/react-dom"
```

Comment:
what the heck do these resolve to without this?

Reply:
well, that's the fun thing: when the react-sdk references React[DOM] itself, such as the case when you have it `yarn link`ed, it puts two React[DOM]s in your bundle. This tells it to use one.